libs/design-system/content-templates/api-command/src/api-command.component.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../../styles/vars' as vars;
+@use '@angular/material' as mat;
+
+:host {
+  @include mat.card-overrides(
+    (
+      elevated-container-shape: 0.25rem,
+      elevated-container-color: vars.$surface-container,
+      elevated-container-elevation: 0,
+    )
+  );
+
+  display: flex;
+  gap: 0.5rem;
+  margin-top: 1.5rem;
+  align-items: center;
+
+  .url-card {
+    border-left: solid #bad2e7 0.5rem;
+    height: 3rem;
+    justify-content: center;
+    overflow: hidden;
+
+    &.post {
+      border-color: #8dc599;
+    }
+  }
+
+  hra-code-block {
+    border: none;
+    padding: 0;
+
+    ::ng-deep code {
+      color: vars.$primary;
```

Comment:
Check whether this color should be the default for all code blocks and if so update the hra-code-block component instead

Reply:
@LibbyUX Do we want to keep the colored text in code blocks or make everything the primary color from now on?

Original issue: https://github.com/hubmapconsortium/hra-ui/issues/911